ci/ray_ci/tester.py

```diff
@@ -40,6 +40,12 @@
     type=str,
     help=("Except tests with the given tags."),
 )
+@click.option(
+    "--include-tags",
```

Comment:
feel that this should just be called `--tags`.. or maybe `--only-tags`. 

`--include-tags` feels to me like: in addition to what is selected (and excluded) also (always) include these tags